CMakeLists.txt

```diff
@@ -3,6 +3,9 @@ project(acquire-driver-zarr)
 cmake_policy(SET CMP0079 NEW) # allows use with targets in other directories
 enable_testing()
 
+find_package(nlohmann_json CONFIG REQUIRED)
+find_package(blosc CONFIG REQUIRED)
```

Comment:
The `CONFIG` keyword is used to specify that the package should be found using a configuration file. CONFIG mode aligns with modern CMake where libraries and packages are expected to provide their own configuration files.